code/modules/worldgen/prefabs.dm

```diff
@@ -179,6 +179,14 @@ ABSTRACT_TYPE(/datum/generatorPrefab)
 		prefabPath = "assets/maps/prefabs/prefab_von_ricken.dmm"
 		prefabSizeX = 42
 		prefabSizeY = 40
+
+	candy_shop //Ryn's store from out of time and out of place
+		maxNum = 1
+		probability = 60 //for initial release, reset to 30 or 25 after a week or so
```

Comment:
please just set this to an appropriate prob, having launched several prefabs this won't negatively impact hype.

Reply:
Will do

Reply:
Done